packages/db/src/translate/pdrs.ts

```diff
@@ -0,0 +1,44 @@
+import Knex from 'knex';
+
+import { ApiPdr } from '@cumulus/types/api/pdrs';
+
+import { CollectionPgModel } from '../models/collection';
+import { ExecutionPgModel } from '../models/execution';
+import { ProviderPgModel } from '../models/provider';
+import { PostgresPdr } from '../types/pdr';
+
+export const translatePostgresPdrToApiPdr = async (
+  postgresPDR: PostgresPdr,
+  knex: Knex | Knex.Transaction,
+  collectionPgModel = new CollectionPgModel(),
+  executionPgModel = new ExecutionPgModel(),
+  providerPgModel = new ProviderPgModel()
+): Promise<ApiPdr> => {
+  const collection = await collectionPgModel.get(knex, {
+    cumulus_id: postgresPDR.collection_cumulus_id,
+  });
+  const execution = await executionPgModel.get(knex, {
```

Comment:
I think this should be conditional because `execution_cumulus_id` is not a required property of a PostgreSQL PDR, so it may not exist. And if it doesn't exist, then this would throw an error I believe?

Reply:
https://github.com/nasa/cumulus/pull/2273/commits/8ba0c6ae4c1b80cdfd5e79079bae29ddcd64d26b